podvm/Makefile.inc

```diff
@@ -11,6 +11,12 @@ UBUNTU_RELEASE     = focal
 SKOPEO_VERSION     = 1.5.0
 UMOCI_VERSION      = 0.4.7
 
+ifeq ($(FORCE),false)
```

Comment:
could you explain this `force` change? I don't understand why it is required for this change, sorry if I am missing something

Reply:
sure, AFAIU force is used to force rebuilding of the forwarder and agent, in the container builds this results in double build, once by `make binaries` and later by `make image` while nothing has changed and process is fixed and always running on clean environment. Using FORCE allows to "unforce" these targets by setting FORCE=false

Reply:
Would this PR have the same result without the force changes and just adding the `clean_sources` target?

In general I'd prefer the pattern of `make clean image` if an end user wanted to force a rebuild. For me doing `make binaries` followed by `make image` doesn't rebuild the binaries already.

Reply:
The target are forced, thing is, if you don't do clean_sources target will be forced but the targets in the source folder makefiles will be skipped
I'll try to do something else

Reply:
I removed the FORCE thing and changed it to a single stage build and added the clean_sources